src/sidebars/version-directive.js

```diff
@@ -0,0 +1,12 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.version')
+        .value('version', '0.1.0')
```

Comment:
What's the rationale behind this number? And do we plan on the release grunt task to automate incrementing it, or will it be manual?